lib/cms/repo.ex

```diff
@@ -328,6 +394,7 @@ defmodule CMS.Repo do
   Paragraphs are stand-alone partials from the CMS. Supports redirects.
   """
   @spec get_paragraph(String.t(), map) :: Paragraph.t() | {:error, any()}
+  @decorate cacheable(cache: @cache, on_error: :nothing, opts: [ttl: @ttl])
```

Comment:
Not sure this is needed since you also decorate the relevant function clause in the `view_or_preview/2`